jep-014-string-functions.md

```diff
@@ -40,30 +40,34 @@ Contrary to similar functions found in most popular programming languages, the `
 | `"subject string"` | `` find_first(@, 'string', `8`) `` |  `8`
 | `"subject string"` | `` find_first(@, 'string', `9`) `` |  `null`
 | `"subject string"` | `` find_first(@, 's') `` |  `0`
-| `"subject string"` | `` find_first(@, 's', 1) `` |  `8`
+| `"subject string"` | `` find_first(@, 's', `1`) `` |  `8`
 | `"subject string"` | `` find_first(@, '') `` |  `null`
 
 ### find_last
 
 ```
-int find_last(string $subject, string $sub[, int $pos])
+int find_last(string $subject, string $sub[, int $start[, $end]])
 ```
 Given the `$subject` string, `find_last()` returns the zero-based index of the last occurence where the `$sub` substring appears in `$subject` or `null` if it does not appear.
```

Comment:
Both `find_first` and `find_last` should document the behavior of an empty-string $sub.

Reply:
I‘ve added the following precision:

_If either the `$subject` or `$sub` argument is an empty string, `find_first()` returns `null`._